### PR TITLE
CR-1232 CR-1234 fixed some layout inconsitencies

### PR DESCRIPTION
**Pull request by tiborpacalat**
Dev: fixed different spacing and different "show more" in Survey Settings -> Overview -> Text elements

### CODE REVIEW

**Comment by tiborpacalat** [comment]
Not sure why the test is failing, didn't touch that piece of functionality.

File: application/views/admin/survey/subview/surveydashboard/002-surveytextpanel.twig [hunk 1]
```diff
@@ -15,14 +15,13 @@
                 <div class="col-4 card-label">
                     {{gT("Description:")}}
                 </div>
-                <div class="col-8 ">
+                <div class="col-8 py-1">
```
Comment (ptelu) [inline application/views/admin/survey/subview/surveydashboard/002-surveytextpanel.twig:18]: should probably added to the parent div and for all items

File: application/views/admin/survey/subview/surveydashboard/004-generalsettingspanel.twig [hunk 2]
```diff
@@ -41,7 +41,7 @@
                         {% if checkPermission('templates','read') %}
                             {% set sTemplateOptionsUrl = createUrl("themeOptions/updateSurvey", {'surveyid' : oSurvey.sid} ) %}
                             {% set sTemplateEditorUrl = createUrl("admin/themes/sa/view", {'templatename' : oSurvey.aOptions['template'] } ) %}
-                            {{templateModel.title}} ( {{templateModel.name}})
+                            {{templateModel.title}} ( {{templateModel.name}} )
```
Comment (ptelu) [inline application/views/admin/survey/subview/surveydashboard/004-generalsettingspanel.twig:44]: i didnt see any difference here, what was the goal?

Comment (tiborpacalat) [reply application/views/admin/survey/subview/surveydashboard/004-generalsettingspanel.twig:44]: Right bracket didn't have space before it.

Comment (tiborpacalat) [reply application/views/admin/survey/subview/surveydashboard/004-generalsettingspanel.twig:44]: Fruity Theme ( fruity)

Comment (ptelu) [reply application/views/admin/survey/subview/surveydashboard/004-generalsettingspanel.twig:44]: aha i thought the issue was aosomething different :) with the positionting of the button at the end not being eligned

**Pull request merged**